### DISCUSSION TITLE
Using a local unregistered package

### DISCUSSION THREAD

**Discussion by yha** [fonsp/Pluto.jl#2235] [Questions about using Pluto]
I'm trying to use a local unregistered package in a notebook. 
One way to do it is using an environment outside the notebook as described here https://github.com/fonsp/Pluto.jl/wiki/%F0%9F%8E%81-Package-management#pattern-the-shared-environment, but I would prefer to have the environment embedded in each notebook, except for the need for a copy of the local package. Is there a way to achieve this?
(I thought I can do it using `Pluto.activate_notebook_environment`. But it seems that if I don't manually manage packages in the notebook, any changes made this way from the REPL are immediately overwritten by Pluto's package manager.)

**Comment by disberd** [comment]
#2245 would allow this if/when implemented
